test/functionalTest/cases/2948_wrong_max_one_service_path_response/wrong_max_one_service_path_response.test

```diff
@@ -0,0 +1,78 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Wrong "max one service-path allowed for subscriptions" in NGSIv2 subscription operation
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create subscription for E1 with multiple service paths header
+
+#
+
+echo "01. Create subscription for E1 with multiple service paths header"
+echo "=============================================================="
```

Comment:
A few more '='s, to be as long as the preceding line

Reply:
Also, the description is not very correct.
Suggestion:
```
Attempt to create subscription for E1 with multiple service paths header, and see it fail
```

Reply:
Fixed in 7480bca